common/types/hashes.go

```diff
@@ -16,18 +16,18 @@ import (
 const (
 	// Hash32Length is 32, the expected length of the hash.
 	Hash32Length = 32
-	hash20Length = 20
-	hash12Length = 12
+	Hash20Length = 20
+	Hash12Length = 12
```

Comment:
unrelated change

Reply:
Oh, yeah I thought i needed the Hash20Length const somewhere but ended up not needing it.
I don't mind reverting, but they do seem like things that should be exported.